content/en/docs/guides/opentelemetry-metrics.md

```diff
@@ -0,0 +1,113 @@
+---
+title: OpenTelemetry Metrics
+description: >-
+  OpenTelemetry Metrics available in gRPC
+---
+
+### Overview
+
+gRPC provides in-built support for an OpenTelemetry plugin that provides metrics that can help you troubleshoot your system, iterate on improving system performance and setup continuous monitoring and alerting.
+
+### Background
+
+OpenTelemetry is an observability framework to create and manage telemetry data. gRPC previously provided observability support through OpenCensus which has been [sunsetted](https://opentelemetry.io/blog/2023/sunsetting-opencensus/) in the favor of OpenTelemetry.
+
+### Client Per-Attempt Instruments
+
+| Name | Type | Unit | Labels (required) | Description |
```

Comment:
The name columns has such large values that it stretches out the table in a funny way. Can you try adding line breaks in the last two names?

Reply:
done